geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloFeatureSource.scala

```diff
@@ -228,7 +232,7 @@ class DelegatingResourceInfo(source: SimpleFeatureSource) extends ResourceInfo {
 
   import scala.collection.JavaConversions._
 
-  private val keywords = Collections.unmodifiableSet(Set("features", getName))
+  private val keywords: java.util.Set[String] = new java.util.HashSet[String](List("features", getName))
```

Comment:
as a defensive programming strategy, it might be better to keep this as an unmodifiable set, and just add source.getSchema.getKeywords when it's created (instead of line 103 above)